Example/saltedge-ios/View Controllers/ProvidersViewController.swift

```diff
@@ -27,25 +27,55 @@ class ProvidersViewController: UIViewController {
         super.viewDidLoad()
         tableView.delegate = self
         tableView.dataSource = self
-        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "ProviderCell")
         searchController.searchResultsUpdater = self
         searchController.hidesNavigationBarDuringPresentation = false
         searchController.dimsBackgroundDuringPresentation = false
         tableView.tableHeaderView = searchController.searchBar
         
+        fetchProviders()
+    }
+    
+    
+    
+    private func fetchProviders() {
         HUD.show(.labeledProgress(title: "Fetching Providers", subtitle: nil))
         SERequestManager.shared.getProviders { [weak self] response in
             switch response {
             case .success(let value):
                 HUD.hide(animated: true)
-                self?.providers = value.data
-                self?.filteredProviders = value.data
-                self?.tableView.reloadData()
+                guard let self = self else { return }
```

Comment:
By our internal convention, we add prefix `weak` to weak variables. (e.g. `guard let weakSelf`)